### PR TITLE
add dangerous-globals-use rule

### PR DESCRIPTION
**Pull request by inkz**
from: https://github.com/returntocorp/enterprise/issues/586

### CODE REVIEW

File: python/lang/security/dangerous-globals-use.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+rules:
+- id: dangerous-globals-use
+  patterns:
+  - pattern-either:
+    - pattern: globals().get(...)
+    - pattern: locals().get(...)
```
Comment (minusworld) [inline python/lang/security/dangerous-globals-use.yaml:6]: `.get(...)` is a helper method for dictionary access - we'll also need to capture `globals()[...]` which does the same thing

Comment (minusworld) [reply python/lang/security/dangerous-globals-use.yaml:6]: Like this: 

```
>>> locals()['__builtins__'].__dict__['abs'](-12)
12
```

Comment (minusworld) [reply python/lang/security/dangerous-globals-use.yaml:6]: Otherwise, looks good! 👍

Comment (inkz) [reply python/lang/security/dangerous-globals-use.yaml:6]: Oh yeah, my bad! :+1:

Comment (inkz) [reply python/lang/security/dangerous-globals-use.yaml:6]: done!

**Review by colleend — APPROVED** [review]
lgtm!!

**Pull request merged**